.gitignore

```diff
@@ -1,10 +1,3 @@
 build/
 .project
-/WebRoot/js/zimbra/
```

Comment:
Actually it is removed, as now ant clean will cleanup these files, it was not doing that earlier

So thought that it would be better to remove it